client/src/components/History/History.jsx

```diff
@@ -1,13 +1,6 @@
 import React from 'react';
-import {
-  Route,
-  Switch,
-  Redirect,
-  useRouteMatch,
-  NavLink as RRNavLink,
-} from 'react-router-dom';
+import { Route, Switch, Redirect, useRouteMatch } from 'react-router-dom';
 import { useSelector, useDispatch } from 'react-redux';
-import { Nav, NavItem, NavLink } from 'reactstrap';
```

Comment:
i wonder if we can add a rule/plugin in our linter/prettier to enforce not using unused imports. follow-on ticket?

Reply:
👍 
https://jira.tacc.utexas.edu/browse/FP-1945